proto/v1/event.proto

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+option go_package = "./proto";
+package datacommons.v1;
+
+message EventCollection {
+  message GeoLocation {
+    message Point {
+      optional double latitude = 1;
+      optional double longitude = 2;
+    }
+    message Polygon { optional string geo_json = 1; }
+    oneof geo {
+      Point point = 1;
+      Polygon polygon = 2;
+    }
+  }
+
+  message PropVal {
+    optional string prop = 1;
+    repeated string val = 2;
+  }
+
+  message Event {
+    // Data Commons places associated with the event.
+    repeated string places = 1;
+    // Geospatial location of the event.
+    repeated GeoLocation geo_locations = 2;
+    // NOTE: The format should be ISO 8601.
+    repeated string dates = 3;
+    repeated PropVal salient_pvs = 4;
```

Comment:
what is salient_pvs?

Reply:
Added comment.

Reply:
are we calling it salient_pvs because it is not all the pv's attached to the event?

Reply:
Right, not all properties are selected, "salient_pv" is a name suggested by @pradh , who might have more sayings here.

Reply:
Yes, not all properties in an event are in the cache, just the more meaningful ones (for example, there are things like some internal IDs, date of review, etc)

I do recognize that this distinction could be useful in the implementation-side (cache proto), but confusing to the user (api proto).  If there's a benefit to keeping the protos the same, we can call it properties (and add a comment in the cache_data.proto).